apis/authentication/v1alpha1/openidconnect_webhook.go

```diff
@@ -109,5 +112,24 @@ func (r *OpenIDConnect) validate() field.ErrorList {
 		}
 	}
 
+	if len(r.Spec.JWKS.Keys) > 0 {
+		if err := validateJWKS(r.Spec.JWKS.Keys); err != nil {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("keys"), string(r.Spec.JWKS.Keys), "must be a valid base64 encoded JWKS"))
+		}
+	}
 	return allErrs
 }
+
+func validateJWKS(jwks []byte) error {
+	data, err := base64.StdEncoding.DecodeString(string(jwks))
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal([]byte(data), &jose.JSONWebKeySet{})
```

Comment:
```suggestion
	err = json.Unmarshal(jwks, &jose.JSONWebKeySet{})
```

I think this should be sufficient.